converter/composer.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I think it's smart that you converted them programmatically, but we don't really need that code after they've been successfully converted. This repo should be for blog posts only 👍